ui/src/app/pages/homepage/data-access-requirements.tsx

```diff
@@ -326,9 +338,8 @@ const Next = () => <FlexRow style={styles.nextElement}>
   <span style={styles.nextText}>NEXT</span> <ArrowRight style={styles.nextIcon}/>
 </FlexRow>;
 
-const ModuleIcon = (props: {moduleName: AccessModule, completedOrBypassed: boolean}) => {
-  const eligible = true; // TODO RW-7059
-  const {moduleName, completedOrBypassed} = props;
+const ModuleIcon = (props: {moduleName: AccessModule, completedOrBypassed: boolean, eligible?: boolean}) => {
```

Comment:
it's now possible to set a module icon to "ineligible" mode: the circle with a minus